src/lib/validations/sql.js

```diff
@@ -215,6 +215,15 @@ const checks: Checks = {
 		excerpt: 'Siteurl/home options',
 		recommendation: '',
 	},
+	siteHomeUrlLando: {
+		matcher: "'(siteurl|home)',\\s?'([^']+vipdev.lndo.site)'",
+		matchHandler: ( lineNumber, results ) => results[ 2 ],
+		outputFormatter: requiredCheckFormatter,
+		results: [],
+		message: 'Siteurl/home options pointing to *.vipdev.lndo.site domain',
+		excerpt: 'Siteurl/home options pointing to lando domain',
+		recommendation: 'Use search-replace to change environment\'s domain',
```

Comment:
@pschoffer how is the recommendation exposed? Should we include the example here like `--search-replace=prod-domain,local-dev-env.vipdev.lndo.site`? Since we likely know both the prod domain (assuming users use `@ID` notation and the the dev env target domain

Reply:
Yeah I tried to get something like that, but the issue is that this validation mechanism is shared with regular SQL import and I would have to build a bit of custom outputFormatter for  just that one. 

I'll think about it some more.